public/app/features/trails/ActionTabs/BreakdownScene.tsx

```diff
@@ -125,13 +129,26 @@ export class BreakdownScene extends SceneObjectBase<BreakdownSceneState> {
     const { labels, body, loading, value } = model.useState();
     const styles = useStyles2(getStyles);
 
+    const useHorizontalLabelSelector = labels.length <= 6;
```

Comment:
wonder if we can use a combination of length + actual label length here as a better heursitic? labels are usually pretty short and I feel the select is sooo much worse here than the radio button so showing the radio button whenever possible is so much better.

Reply:
How about for now, we make it 9 instead of 6, and then in a separate PR we consider the width.

Reply:
>  actual labels text length

I will take a look at this first, actually.